README.md

```diff
@@ -29,23 +31,65 @@ Installs Splunk or Universal Forwarder.
+| base_url |               String                |    No    | `'https://download.splunk.com/products'`                                  | Base url to pull Splunk packages from. Use this if you are mirroring the downloads for Splunk packages. The resource will append the version, os, and filename to the url like so: `{base_url}/splunk/releases/0.0.0/linux/splunk-0.0.0-a1b2c3d4e5f6-Linux-x86_64.tgz` |
 
 ##### Action *:uninstall*
 Removes Splunk or Universal Forwarder and all its configuration.
 
 Properties:
 
-| Name    |               Type(s)               | Required | Default | Description                                                                                                                                                                                         |
-|:--------|:-----------------------------------:|:--------:|:--------|:----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| package | `:splunk` or `:universal_forwarder` | **Yes**  |         | Specifies the Splunk package to uninstall. You must specify the package, or name the resource for the package; for example, `package :splunk` or `splunk_install 'universal_forwarder' do ... end`. |
+| Name    |               Type(s)               | Required | Default | Description                                                                                                                                                                                        |
+|:--------|:-----------------------------------:|:--------:|:--------|:---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| package | `:splunk` or `:universal_forwarder` | **Yes**  |         | Specifies the Splunk package to uninstall. You must specify the package, or name the resource for the package; for example, `package :splunk` or `splunk_install 'universal_forwarder' do ... end` |
 
 
+### splunk_service
+Manages an installation of Splunk
+*Note: Multiple installations of Splunk may technically be possible, but currently only one can be enabled as a system service.*
+
+**The following actions share the same properties:**
+
+##### Action *:start*
+Starts the Splunk daemon if not already running.
+If the ulimit is changed, invokes a restart of the daemon at the end of the run.
+
+##### Action *:restart*
+Starts the Splunk daemon, stopping it if it is already running.
```

Comment:
Minor, but this seems a weird way to phrase this. Might be better like: `Stops the Splunk daemon, if it is running, and starts it again.`